ui/app/templates/vault/cluster/access/identity/index.hbs

```diff
@@ -43,9 +43,7 @@
                   </li>
                   {{#if (or item.isReloading item.updatePath.isPending item.aliasPath.isPending)}}
                     <li class="action">
-                      <button disabled type="button" class="link button is-loading is-transparent">
```

Comment:
Went ahead and replaced these with new component even though they were out of my jurisdiction

Reply:
Lol "jurisdiction" great word choice 😂